subsys/caf/modules/sensor_manager.c

```diff
@@ -385,52 +385,53 @@ static void configure_max_power_state(void)
 	}
 }
 
-static int sensor_init(void)
+static size_t sensor_init(void)
 {
-	int err = 0;
+	size_t alive_sensors = 0;
 	int64_t cur_uptime = k_uptime_get();
 
-	for (size_t i = 0; !err && (i < ARRAY_SIZE(sensor_data)); i++) {
+	for (size_t i = 0; i < ARRAY_SIZE(sensor_data); i++) {
 		struct sensor_data *sd = &sensor_data[i];
 		const struct sm_sensor_config *sc = &sensor_configs[i];
 
 		sd->dev = device_get_binding(sc->dev_name);
 		if (!sd->dev) {
-			err = -ENXIO;
 			update_sensor_state(sc, sd, SENSOR_STATE_ERROR);
-			break;
+			LOG_ERR("%s sensor failed to initialize", sc->dev_name);
```

Comment:
Maybe:
`%s - Cannot get sensor binding`
`%s - Cannot initialize trigger`
So that user would be aware about root cause (you have 2 separate logs anyway)